packages/server/src/fhir/routes.ts

```diff
@@ -97,6 +98,9 @@ protectedRoutes.get('/Group/:id/([$]|%24)export', asyncWrap(groupExportHandler))
 // Bulk Data
 protectedRoutes.use('/bulkdata', bulkDataRouter);
 
+// Async Job
+protectedRoutes.use('/AsyncJob', asyncJobRouter);
```

Comment:
Hmm, we should talk about this one.  This will prevent us from being able to use any of the normal FHIR semantics for the `AsyncJob` resource, because it has the same name as the resource type.  Was that intended?

Reply:
I've renamed the route to `/fhir/R4/job/{{AsyncJob.id}}/status
` to avoid possible conflicts.